Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -0,0 +1,140 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import ComponentLibrary
+import UIKit
+import Shared
+
+class FakespotViewController: UIViewController, Themeable {
+    private struct UX {
+        static let closeButtonWidthHeight: CGFloat = 30
+        static let topLeadingTrailingSpacing: CGFloat = 18
+        static let logoSize: CGFloat = 36
+        static let titleLabelFontSize: CGFloat = 17
+        static let headerSpacing = 8.0
+        static let headerBottomMargin = UIEdgeInsets(top: 0, left: 0, bottom: 8, right: 0)
+    }
+
+    var notificationCenter: NotificationProtocol
+    var themeManager: ThemeManager
+    var themeObserver: NSObjectProtocol?
+
+    private lazy var scrollView: UIScrollView = .build()
```

Comment:
Pretty cool we can call it with `()`, I've seen in other places like `.build { _ in }` which is not that great.